File: cardbattlemodule/src/main/java/com/syahputrareno975/cardbattlemodule/interfaces/LobbyStreamEventController.java
package com.syahputrareno975.cardbattlemodule.interfaces;

import com.syahputrareno975.cardbattlemodule.model.player.PlayerModel;

public interface LobbyStreamEventController {
    void getAllPlayer();
    void getOnePlayer(String id);
    void getMyPlayerData(String id);
    void leftGame(PlayerModel p,SimpleUnit action );
    void leftLobby(SimpleUnit action);
}

File: app/src/main/java/com/syahputrareno975/simplecardbattle/interfaces/LobbyStreamEventController.kt
package com.syahputrareno975.simplecardbattle.interfaces

import com.syahputrareno975.simplecardbattle.model.card.CardModel
import com.syahputrareno975.simplecardbattle.model.player.PlayerModel

interface LobbyStreamEventController {
    fun getAllPlayer()
    fun getOnePlayer(id : String)
    fun getMyPlayerData(id : String)
    fun leftGame(p : PlayerModel,action :() -> Unit)
    fun leftLobby(action : () -> Unit)
}